lib/mix/lib/mix/tasks/deps.clean.ex

```diff
@@ -73,6 +73,14 @@ defmodule Mix.Tasks.Deps.Clean do
     apps -- Enum.map(deps, &Atom.to_string(&1.app))
   end
 
+  defp invalid_path_warning?(path, dependency) do
```

Comment:
We shouldn't call this something that ends in `?` as by convention that means that a function returns a boolean. I think a good name for this function would be `maybe_warn_for_invalid_path/2`, wdyt?